de.dashup.shared/src/main/java/de/dashup/shared/DatabaseModels/DatabaseObject.java

```diff
@@ -0,0 +1,7 @@
+package de.dashup.shared.DatabaseModels;
```

Comment:
In Java packages are written in lower case.

Reply:
You are right, my fault.